app/views/trips/_import_form.html.haml

```diff
@@ -1,7 +1,20 @@
+<br>
```

Comment:
any reason not using haml?

Reply:
I use html and haml interchangeably, I'll fix it

Reply:
Prefer using haml if the file is not already erb. There are certain things that _are_ more ergonomic in regular html, but that's not the case here.